app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -0,0 +1,321 @@
+import { request } from '@utilities/http';
+
+function addCloseListener() {
+  const button = document.querySelector('.close-actions-panel');
+  button.addEventListener('click', () => {
+    // getting the article show page document because this is called within an iframe
+    const articleDocument = top.document;
+
+    articleDocument
+      .querySelector('.mod-actions-menu')
+      .classList.toggle('showing');
+    articleDocument
+      .querySelector('.mod-actions-menu-btn')
+      .classList.toggle('hidden');
+  });
+}
+
+function initializeHeight() {
+  document.documentElement.style.height = '100%';
+  document.body.style.cssText = 'height: 100%; margin: 0; padding-top: 0;';
```

Comment:
Yeah, so because the actions panel is loaded via iframe we need to do some JS + CSS hacking here to configure the height of the page. Otherwise the actions panel doesn't take up the full page of the article show page.